buckaroo3.php

```diff
@@ -320,6 +322,16 @@ public function hookPaymentOptions($params)
                 $phone_mobile_billing = $address['phone_mobile'];
                 $firstNameBilling = $address['firstname'];
                 $lastNameBilling = $address['lastname'];
+                $address1 = $address['address1'];
+            }
+        }
+        $houseNumber = null;
+        //Trim spaces
+        $trimAddress = preg_replace('!\s+!', ' ',$address1);
```

Comment:
to get the house number we use a helper function in most of our plugin see: https://github.com/buckaroo-it/WooCommerce/blob/master/library/Buckaroo_Order_Details.php#L152

Reply:
That function will only recognize the house number if it is in specific place in the address in this way it will be the first numeric string which I think makes more sense